submissions/EmmanuelIdiaghe/php-database/admin.php

```diff
@@ -0,0 +1,86 @@
+<?php
+#Variable Declaration
+$notSet = !(isset($_COOKIE["show"])); //cookie not set
+
+# Function that prints in the console (like console.log)
+function debug_to_console($data) {
+    $output = $data;
+    if (is_array($output))
+        $output = implode(',', $output);
+
+    echo "<script>console.log('Debug Objects: " . $output . "' );</script>";
+}
+
+# Function that sets cookie
+function load() {
+    //SetUp Connection
+    global $pdo;
+    $usernameDB = "dufuna";//user
+    $passwordDB = "dufuna12345";//user password
+
+    try {
+        $pdo = new PDO("mysql:host=localhost", $usernameDB, $passwordDB);
+        $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+        echo "Connected Successfully. Host info: " . $pdo->getAttribute(constant("PDO::ATTR_CONNECTION_STATUS")) . "<br>";
+    
+    } catch (PDOException $e) {
+        die ("ERROR: Could not connect. " . $e->getMessage()) . "<br>";
+    }
+    
+    #Use Database
+    try {
+        $sql = "USE emmy_shop";
+        $pdo->exec($sql);
+        echo "Using Database: emmy_shop <br><hr><br>";
+    
+    } catch (PDOException $e) {
+        die ("ERROR: Could not use database $sql " . $e->getMessage()) . "<br>";
+    }
+    
+    $sql = "SELECT * FROM customers";
+    $result = $pdo->query($sql);
+    $details = $result->fetchAll();
+    setcookie("show", serialize($details), time() + (86400 * 30), "/");
+    $_COOKIE["show"] = serialize($details);
+}
+
+
+try {
+    if ($notSet) {
+        load(); //set cookie
+        //debug_to_console($details);
```

Comment:
Remove all the debugging you did. You don't want to leave your debugging imagining you are finally going to production environment

Reply:
Yeah, true!
I've done that!